apps/load-generator/main.py

```diff
@@ -112,14 +144,21 @@ def main():
         Scaler(config["scaler"], url, req_kwargs).run()
         return
 
-    hist = Histogram("loadgen_query_duration_seconds", "Query duration", 
-            ["prometheus", "group", "query"],
-            buckets=(0.01, 0.05, 0.1, 0.3, 0.7, 1.5, 3, 6, 12, 18, 28))
+    queryTimeHist = Histogram("loadgen_query_duration_seconds", "Query duration",
+        ["prometheus", "group", "queryString", "queryType", "start", "end", "step"],
+        buckets=(0.01, 0.05, 0.1, 0.3, 0.7, 1.5, 3, 6, 12, 18, 28))
+    totalQueriesCnt = Counter('loadgen_total_queries', 'Total amount of queries',
+        ["prometheus", "group", "queryString", "queryType", "start", "end", "step"],
+    )
```

Comment:
Counters always should end in `_total`. So `loadgen_queries_total` and `loadgen_failed_queries_total`.